src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -452,13 +452,25 @@ public void testProxyConnectionWithoutProxyPortOrHost()
             + "&proxyUser=testuser1&proxyPassword=test"
             + "&useProxy=true";
     try {
-      runAzureProxyConnection(connectionUrl, false);
+      runAzureProxyConnection(connectionUrl, false, true);
       fail();
     } catch (SQLException e) {
       assertEquals(SqlState.CONNECTION_EXCEPTION, e.getSQLState());
     }
   }
 
+  @Test
```

Comment:
Add test to ensure PUT/GET now go through proxy

Reply:
Does this test mean that we have to check the test log manually and make sure the put/get command go through the proxy? If so, can we add some comments?

Reply:
Good point! This test should be run with a Burpsuite proxy, which allows the user to see what requests are passing through the proxy. I added test comments above. There are also comments at the top of CustomProxyLatestIT, so I added a link in those comments to the Confluence page detailing how to set up Burpsuite for future users.